resources/language/resource.language.nl_nl/strings.po

```diff
@@ -0,0 +1,166 @@
+# Service Up Next language file
+# Addon Name: Up Next
+# Addon id: service.upnext
+# Addon Provider: im85288
+msgid ""
+msgstr ""
+"Project-Id-Version: service.upnext\n"
+"Report-Msgid-Bugs-To: http://trac.xbmc.org/\n"
+"POT-Creation-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"PO-Revision-Date: 2018-02-26 13:31+0000\n"
+"Last-Translator: dagwieers\n"
+"Language-Team: Dutch\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: nl\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 1.8.7\n"
+
+msgctxt "#30001"
+msgid "The number of seconds before the end to show the notification"
+msgstr "Aantal seconden voor het einde om meldingen te tonen"
+
+msgctxt "#30002"
+msgid "Log Level:"
+msgstr "Log niveau"
+
+msgctxt "#30003"
+msgid "Settings"
+msgstr "Instellingen"
+
+msgctxt "#30004"
+msgid "Advanced"
+msgstr "Geavanceerd"
+
+msgctxt "#30005"
+msgid "Default action when nothing selected"
+msgstr "Standaard actie als niets geselecteerd is"
+
+msgctxt "#30006"
+msgid "Watch Now"
+msgstr "Bekijk nu"
+
+msgctxt "#30008"
+msgid "Up Next"
+msgstr "Volgend"
+
+msgctxt "#30009"
+msgid "Number of episodes before still watching query"
+msgstr "Aantal afleveringen alvorens te vragen of er nog gekeken wordt"
+
+msgctxt "#30010"
+msgid "Continue watching"
+msgstr "Verder kijken"
+
+msgctxt "#30013"
+msgid "Include watched episodes"
+msgstr "Toepassen op bekeken afleveringen"
```

Comment:
Eventueel: Tel bekeken afleveringen mee

Reply:
Ik had moeite om "Include" hier te vertalen, dus ik vond "toepassen" wel goed gevonden.
Meetellen vind ik in dit geval vreemder. Ik twijfel ook altijd tussen de volgende:
- Pas op bekeken afleveringen toe
- Toepassen op bekeken afleveringen
- Op bekeken afleveringen toepassen

Eigenlijk klinkt het derde beter hier.

Reply:
Vandaar dat ik eventueel schrijf, het is inderdaad lastig vertalen, een wijziging is niet per se nodig. Dus kies zelf maar.

Reply:
Ik heb er van gemaakt:
- Op bekeken afleveringen toepassen
maar het zou ook kunnen worden:
- Ook op bekeken afleveringen toepassen

Reply:
Ik heb het laatste gekozen, en ook de andere items zo aangepast: werkwoord als infinitief.
Ik herinner me dat @dnicolaas dat ook zo deed voor inputstreamhelper.